From 90c53495fe04943b0c82c723df423d220dfb408c Mon Sep 17 00:00:00 2001
Message-Id: <patch-1.2-90c53495f-20221223T174835Z-avarab@gmail.com>
In-Reply-To: <cover-0.2-000000000-20221223T174835Z-avarab@gmail.com>
References: <cover-0.2-000000000-20221223T174835Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 23 Dec 2022 18:32:47 +0100
Subject: [PATCH 1/2] proc.5: note that "cmdline" might be favored over
 "stat.comm" by ps(1)
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

With "ps -A" the output uses the "stat.comm" field, but "ps a" will
display the value in "cmdline".

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 man5/proc.5 | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/man5/proc.5 b/man5/proc.5
index 65a4c38e3..115c8592e 100644
--- a/man5/proc.5
+++ b/man5/proc.5
@@ -2087,7 +2087,11 @@ The affected fields are indicated with the marking [PT].
 The process ID.
 .TP
 (2) \fIcomm\fP \ %s
-The filename of the executable, in parentheses.
+The filename of the executable, in parentheses. Tools such as
+.BR ps (1)
+may alternatively (or additionally) use
+.IR /proc/ pid /cmdline.
+.IP
 Strings longer than
 .B TASK_COMM_LEN
 (16) characters (including the terminating null byte) are silently truncated.
-- 
2.39.0.1106.gf45ba805d1a

